rules.md

```diff
@@ -395,9 +418,33 @@ In cases where there is a serious threat to life, such as being punched to the b
 - Being punched twice on full health is not a serious threat to your life. It would not warrant you throwing hell to the wind and blasting away your comrades.
 - Being punched twice after already being critically injured can be seen as a risk to your life, and you can take appropriate action. This is subject to circumstances however and not a greenlight for any situation.
 
+## Jailbreak
+
+Escalation still applies when attempting to remove a marine from the brig after being imprisoned by the Military Police.
+
+For example:
+- You cannot use C4 or lethal explosives in order to break into the brig and release a marine.
+- You can properly escalate fights with MP as outlined previously, however, this will still result in IC sanctions.
+  - MP tools are not considered lethal weaponry. Unless an MP is shooting you, or using a baton on harm intent, you can't respond lethally.
+
+## Battlefield Executions
+
+- Whitelisted Commanding Officers are allowed to instantly kill any player at any time if the player fulfills the conditions found in Marine Law. Excessive or poorly-reasoned battlefield executions (such as BEing MPs over a valid arrest) may end with the Commanding Officers being removed from the whitelist.
+
+## Hostile Factions
+
+If you are a member of a faction that is hostile to the USMC, you are not held by Lethal Force rules.
+
+This includes:
+- [Colonial Liberation Front](https://cm-ss13.com/wiki/CLF)
+- [Union of Progressive People](https://cm-ss13.com/wiki/UPP)
```

Comment:
UPP are not inherently hostile, and this makes things a little ambiguous and allows marines to gun them down.

Reply:
That seems totally legit and I'll fix that real fast.